src/Http/Controllers/RemoveController

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/*
+ * This file is part of Starbs Yeh.
+ *
+ * (c) Graham Campbell <graham@mineuk.com>
+ * (c) Michael Banks <chip@mineuk.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Starbs\Yeh\Http\Controllers;
+
+use Starbs\Http\Controllers\AbstractController;
+
+class RemoveController extends AbstractController
+{
+    /**
+     * Do some clever things, then return a response.
+     *
+     * @return \Symfony\Component\HttpFoundation\Response
+     */
+    protected function fire()
+    {
+        $image = $this->container->get('factory')->remove($this->args['id']);
+
+        if (!$image) {
+            return $this->success(['message' => 'Image Deleted'], 410);
```

Comment:
Meh, I'll just merge it into 1.1 and address the with when I've got time. 1.0 is deployed to our server.